test/ops/cwise_ops_binary_test.py

```diff
@@ -854,8 +854,7 @@ def testPowNegativeExponentCpu(self):
   def testPowNegativeExponentGpu(self):
     if not test_util.is_gpu_available():
       self.skipTest("Requires GPU")
-    # TFDML #24881131
-    self.skipTest("DML doesn't support negative int64 numbers yet.")
+    self.skipTest("DML doesn't support negative int64 numbers for pow.")
```

Comment:
Is it only for int64 or for negative numbers of all types?

Reply:
good point :)